src/plugins/dashboard/public/types.ts

```diff
@@ -141,3 +142,19 @@ export interface StagedFilter {
   operator: string;
   index: string;
 }
+
+export interface DashboardListItem {
```

Comment:
Can you consolidate all the new interfaces here?

Reply:
Yes, done.